site2/website/release-notes.md

```diff
@@ -1,6 +1,181 @@
+- Broker expires messages one at a time after topic unload [#9083](https://github.com/apache/pulsar/pull/9083)
+- Add refresh authentication command in broker [#9064](https://github.com/apache/pulsar/pull/9064)
+- Add updateRates method for kop to collect publish rate [#9094](https://github.com/apache/pulsar/pull/9049)
+- Fix DelayedDelivery at the broker level has a default value [#9030](https://github.com/apache/pulsar/pull/9030)
+- Getting the stats of a non-persistent topic that has been cleaned causes it to re-appear [#9029](https://github.com/apache/pulsar/pull/9029)
+- Add raw prometheus metrics provider [#9021](https://github.com/apache/pulsar/pull/9021)
+- Improve error handling when broker doesn't trust client certificates [#8998](https://github.com/apache/pulsar/pull/8998)
+- remove duplicated broker prometheus metrics type [8995](https://github.com/apache/pulsar/pull/8995)
+- Peeking at compressed messages throws an exception (Readonly buffers not supported by Airlift) [#8990](https://github.com/apache/pulsar/pull/8990)
+- Make namespaces isolation policy updates take effect on time [#8976](https://github.com/apache/pulsar/pull/8976)
+- Fix NPE in PersistentStickyKeyDispatcherMultipleConsumers [#8969](https://github.com/apache/pulsar/pull/8969)
+- Fix the recovery not respect to the isolation group settings [#8961](https://github.com/apache/pulsar/pull/8961)
+- Add properties default value for SchemaInfoBuilder [#8952](https://github.com/apache/pulsar/pull/8952)
+- Consumer support update stats with specified stats [#8951](https://github.com/apache/pulsar/pull/8951)
+- Support configure max subscriptions per topic on the topic level policy [#8948](https://github.com/apache/pulsar/pull/8948)
+- Fix subscription dispatch rate does not work after the topic unload without dispatch rate limit [#8947](https://github.com/apache/pulsar/pull/8947)
+- Avro custom schema not working in consumer [#8939](https://github.com/apache/pulsar/pull/8939)
+- Expose non-contiguous deleted messages ranges stats [#8936](https://github.com/apache/pulsar/pull/8936)
+- Intercept beforeSendMessage calls [#8932](https://github.com/apache/pulsar/pull/8932)
+- Monitor if a cursor moves its mark-delete position [#8930](https://github.com/apache/pulsar/pull/8930)
+- Capture stats with precise backlog [#8928](https://github.com/apache/pulsar/pull/8928)
+- Support configure max subscriptions per topic on the namespace level policy [#8924](https://github.com/apache/pulsar/pull/8924)
+- Export Prometheus metric for messageTTL [#8871](https://github.com/apache/pulsar/pull/8871)
+- Add pulsar-perf new feature: one subscription has more than one consumer [#8837](https://github.com/apache/pulsar/pull/8837)
+- Execute removing non-persistent subscription of topic from different thread to avoid deadlock when removing inactive subscriptions [#8820](https://github.com/apache/pulsar/pull/8820)
```

Comment:
```suggestion
- Execute removing non-persistent subscription of a topic from a different thread to avoid deadlock when removing inactive subscriptions [#8820](https://github.com/apache/pulsar/pull/8820)
```